### ISSUE TITLE
(SP: 3) Create Link element aligned with design system

### ISSUE THREAD

**Issue by luvthenika** [ita-social-projects/Liatoshynsky#109]
Custom Link should be both styles and optisimized:

- Next Link from 'next/link' for optimization
- MUI link for styling options 
Important details to implement:

- [ ] can go along with icons
![Image](https://github.com/user-attachments/assets/d0825455-9be5-48e5-8806-d84fa82fcf0c)
- [ ] can be disabled
![Image](https://github.com/user-attachments/assets/fdac9e8c-72b8-4e9a-97a0-f0d5802cbd48)
- [ ]  can be hovered
![Image](https://github.com/user-attachments/assets/1a50d28f-61c9-493d-beb9-a2a33ed4eccd)
